app.js

```diff
@@ -8,42 +10,92 @@ app.use(logger("dev"));
 app.use(express.static("public"));
 app.use(express.json());
 
-
 // Iteration 1 - Connect to MongoDB
 // DATABASE CONNECTION
-
-
+const MONGODB_URI = "mongodb://127.0.0.1:27017/express-mongoose-recipes-dev";
+mongoose
+  .connect(MONGODB_URI)
+  .then((x) =>
+    console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`)
+  )
+  .catch((err) => console.error("Error connecting to mongo", err));
 
 // ROUTES
-//  GET  / route
-app.get('/', (req, res) => {
-    res.send("<h1>Lab | Express Mongoose Recipes</h1>");
+//  GET  / route - This is just an example route
+app.get("/", (req, res) => {
+  res.send("<h1>LAB | Express Mongoose Recipes</h1>");
 });
 
-
-//  Iteration 4 - Create recipe route
+//  Iteration 3 - Create a Recipe route
 //  POST  /recipes route
 
+app.post("/recipes", (req, res) => {
+  Recipe.create(req.body)
```

Comment:
```suggestion
Something that we can  do in order to keep the code easy to read is destructuring the properties from it, for people that do not know the schema at all
```suggestion
const { title, instructions, level, ingredients, image ,duration, isArchived, created} = req.body
  Recipe.create({
    title,
    instructions,
    level,
    ingredients,
    image,
    duration,
    isArchived,
    created,
  })
```
```